classes/submissionFile/Repository.php

```diff
@@ -379,7 +379,7 @@ public function edit(
                 'assocId' => $submission->getId(),
                 'eventType' => $newFileUploaded ? SubmissionFileEventLogEntry::SUBMISSION_LOG_FILE_REVISION_UPLOAD : SubmissionFileEventLogEntry::SUBMISSION_LOG_FILE_EDIT,
                 'message' => $newFileUploaded ? 'submission.event.revisionUploaded' : 'submission.event.fileEdited',
-                'isTranslate' => 0,
+                'isTranslate' => false,
```

Comment:
I see the use of both `isTranslate` and `isTranslated`; can we standardize on one? (`isTranslated` makes more sense to me.)